hibernate-core/src/main/java/org/hibernate/Timeouts.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate;
+
+import jakarta.persistence.Timeout;
+
+/**
+ * Magic {@linkplain jakarta.persistence.Timeout time out} values.
+ *
+ * @apiNote Generally speaking, for {@linkplain #NO_WAIT} and {@linkplain #SKIP_LOCKED},
+ * the corresponding {@linkplain LockMode} should be preferred, but these magic values
+ * allow use in conjunction with {@linkplain jakarta.persistence.LockModeType} without
+ * the need to cast/unwrap to Hibernate specifics.
+ *
+ * @author Steve Ebersole
+ *
+ * @since 7.0
+ */
+public interface Timeouts {
+	/**
+	 * Raw magic millisecond value for {@linkplain #NO_WAIT}
+	 */
+	int NO_WAIT_MILLI = 0;
+
+	/**
+	 * Raw magic millisecond value for {@linkplain #WAIT_FOREVER}
+	 */
+	int WAIT_FOREVER_MILLI = -1;
+
+	/**
+	 * Raw magic millisecond value for {@linkplain #SKIP_LOCKED}
+	 */
+	int SKIP_LOCKED_MILLI = -2;
```

Comment:
Do we really need to expose these here?

Reply:
I've been using them.  But no we really don't - I could also use `Timeouts.SKIP_LOCKED.milliseconds()`, etc

Reply:
Well, that actually messes up swtich-case statements where these are used a lot.

Reply:
I was only suggesting to move them somewhere else. Perhaps to `o.h.dialect` or whatever.

Reply:
Sure.  That feels unnatural to me though